Add giallo zafferano to scrapers, closes #28

marcolussetti opened the pull request:
This Pull Request adds support for a new scraper for the website Giallo Zafferano, Italy's largest recipe site.

1. To parse the instructions, I parsed embedded json with the standard library's json module. It seemed to me a cleaner option than trying to extract the various paragraphs of instructions that had no real markers (could do between tags, but it seemed not very clean and brittle). I can rewrite it to use pure beautifulsoup if preferred.
2. I wrote tests for the code, but I have to admit I'm pretty new to it so I don't know if I went about it the right way. The test clear fine on my machine running against a saved html page obtained via:
```bash
curl -o recipe_scrapers/tests/test_data/giallozafferano.html http://ricette.giallozafferano.it/Bavarese-alle-fragole.html
```

Please let me know if this is all right or if you would like anything changed ;-)

hhursev commented:
This is perfect addition! Also I want to thank you for being extremely careful with the repo's code base. You even included the giallozafferano's link at the right place in the Readme.md file!

I’d like to make this PR a standard for forthcoming PRs. I tidied up the repo for you and made it possible to add tests. Can you please:

- add tests for Giallo Zafferano
- rebase the PR so to stay on top of the master branch (again)
- last is needed as I love fast forward merges

P.P. Love what you did with the instructions! Leave it like that :)

marcolussetti commented:
Thank you for your kind words, it's a very easy repository to work with I think :)

I've added the test data so the tests from earlier should be working fine now and rebased the PR on top of it, hope that works :)

klebinhopk commented:
Poderia me ajudar a rodar esse script do python por favor?
I From Brazil
